src/commands/coverage/utils.ts

```diff
@@ -0,0 +1,70 @@
+import fs, {lstatSync} from 'fs'
+import path from 'path'
+
+import {XMLValidator} from 'fast-xml-parser'
+
+export const isFile = (filePath: string) => {
+  try {
+    return lstatSync(filePath).isFile()
+  } catch (e) {
+    return false
+  }
+}
+
+export const validateCoverageReport = async (
+  filePath: string,
+  format: string | undefined,
+  userProvidedFormat: string | undefined
+) => {
+  if (format === undefined) {
+    if (userProvidedFormat) {
+      format = userProvidedFormat
+    } else {
+      return 'Could not detect format of ' + filePath + ', please specify the format manually using the --format option'
```

Comment:
You can use template strings: 
```suggestion
      return `Could not detect format of ${filePath}, please specify the format manually using the --format option`
```

Reply:
Good point, replaced with template strings